### ISSUE TITLE
Chặn tracking/cookie trên motphim.co (motphim.net)

### ISSUE THREAD

**Issue by vhpcdpgl** [bigdargon/hostsVN#194]
[//]: # (***Vui lòng nhập thông tin phía dưới dòng có dấu "[//]:" các dòng này sẽ bị ẩn khi đăng bài. Xin cám ơn!)

[//]: # (Nếu bạn muốn thêm tên miền mới, vui lòng đưa tên miền vào thẻ bên dưới.)

***Tên miền***:
```
dc.motphim.co
dc.motphim.net
```
Ví dụ script:
```
https://dc.motphim.co/tracking?type=count_views&id=8027&eId=101202
https://dc.motphim.co/tracking?type=count_views&id=7992&eId=100665
```
`https://dc.motphim.co/ajax.html` tạo các cookie cho motphim.co, ví dụ như:
`__cfduid=dcf13a5dca5d0cc2bec5e776776ddeada1590464256; expires=Thu, 25-Jun-20 03:37:36 GMT; path=/; domain=.motphim.co; HttpOnly; SameSite=Lax; Secure`
`__cfduid=de562581dd99fda9d8dced2eec76ac8251590463882; expires=Thu, 25-Jun-20 03:31:22 GMT; path=/; domain=.motphim.co; HttpOnly; SameSite=Lax; Secure`

Chặn xong vẫn xem phim bình thường.
Ngoài ra trang motphim.co còn nạp thông tin từ `dc.motphim.net` , subdomain này ngoài việc tạo cookie còn chứa các hàm API để nhúng QC dạng cửa sổ/frame từ các trang bên ngoài, ví dụ: 
```
https://dc.motphim.net/api/f1
https://dc.motphim.net/api/f2
```
Mình phát hiện cái script của `dc.motphim.net/api/f1` thực ra là frame quảng cáo để load nội dung từ trang `dvdmood.com`:
<iframe src="/api/f1" scrolling="no" border="0" frameborder="0" style="border: 0px; padding: 0px; margin: 0px; width: 300px; height: 250px;">

![dc motphim net 1](https://user-images.githubusercontent.com/59459998/82860179-0264b100-9f43-11ea-8c6b-f1669dd6e1d1.png)
![dc motphim net 2](https://user-images.githubusercontent.com/59459998/82860183-055fa180-9f43-11ea-81e6-b8d34358e2bd.png)

Còn `dc.motphim.net/api/f2` thì  tạo cookie cho motphim.net, ví dụ như:
`__cfduid=dcec2ef2a9fb27fd0a9dd26bff89da5bf1590466142; expires=Thu, 25-Jun-20 04:09:02 GMT; path=/; domain=.motphim.net; HttpOnly; SameSite=Lax`

**Comment by vhpcdpgl** [comment]
Chặn xong vẫn xem phim bình thường.

**Comment by vhpcdpgl** [comment]
Sau khi chặn xong:
![motphim1](https://user-images.githubusercontent.com/59459998/82860901-45c01f00-9f45-11ea-9196-0b26f4ec9eae.jpg)
![motphim2](https://user-images.githubusercontent.com/59459998/82860899-435dc500-9f45-11ea-8f84-3d993d0cbbbc.jpg)

**Comment by vhpcdpgl** [comment]
Bổ sung thêm domain qc:
```
advgalaxy.com
z.cdn.advgalaxy.com
boyaidare.com
```
Source: View page source.

**Comment by bigdargon** [comment]
@vhpcdpgl Tên miền `advgalaxy.com` mình đã thêm vào, còn tên miền `boyaidare.com` không tồn tại bạn kiểm tra lại xem đúng chưa?

**Comment by vhpcdpgl** [comment]
> @vhpcdpgl Tên miền `boyaidare.com` không tồn tại bạn kiểm tra lại xem đúng chưa?

 Giờ chắc là `boyaidare.club` đó bạn. Cái .com là trong source của trang.
